docs/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Codename: Snoopy</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="container">
+      <h1>Codename:<br />Snoopy</h1>
+
+      <p>Immediate feedback on your React components, as you develop.</p>
+
+      <p class="code"><code>npx @prodo-ai/snoopy</code></p>
```

Comment:
I think someone already mentioned yesterday that we're not sure if we should be recommending npx due to slowness. It is more user friendly though so I don't have strong feelings either way.

Reply:
I think @coffee-cup is right: if people run through `npx`, do a sneaky `npm install --save-dev`/`yarn add --dev` for them.